save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DiktatDemoRunRequest.kt

```diff
@@ -10,4 +11,4 @@ import kotlinx.serialization.Serializable
 data class DiktatDemoRunRequest(
     val codeLines: String,
```

Comment:
maybe pass as lines instead of a single string to avoid problem with EOL

Reply:
It makes sense, will do it

Reply:
Done